### PR TITLE
FRI-98 - Added response queue mechanism so RVF states can be sent back to the SRS service.

### PR DESCRIPTION
**Pull request by Tyler-Cooksey**

### CODE REVIEW

File: execution-service/src/main/java/org/ihtsdo/rvf/execution/service/config/ValidationRunConfig.java [hunk 1]
```diff
@@ -134,6 +136,15 @@ public ValidationRunConfig setForm(CharacteristicType form) {
 		return this;
 	}
 
+	public final String getResponseQueue() {
+		return responseQueue;
+	}
+
+	public final ValidationRunConfig setResponseQueue(final String responseQueue) {
+		this.responseQueue = responseQueue;
```
Comment (CoderMChu) [inline execution-service/src/main/java/org/ihtsdo/rvf/execution/service/config/ValidationRunConfig.java:144]: I would rename this method name to addResponseQueue

File: api/src/main/java/org/ihtsdo/rvf/messaging/ValidationQueueManager.java [hunk 2]
```diff
@@ -64,6 +68,9 @@ public void queueValidationRequest(ValidationRunConfig config, Map<String, Strin
 				String configJson = gson.toJson(config);
 				LOGGER.info("Send Jms message to queue for validation config json:" + configJson);
 				jmsTemplate.convertAndSend(destinationName, configJson);
+				messagingHelper.send(config.getResponseQueue(),
```
Comment (CoderMChu) [inline api/src/main/java/org/ihtsdo/rvf/messaging/ValidationQueueManager.java:71]: You need to check that response queue is not null first before sending response message. (i.e sending response messages only when the queue is specified).

**Review by CoderMChu — CHANGES_REQUESTED** [review]
@Tyler-Cooksey It looks good with just minor changes needed. Thanks.

File: execution-service/src/main/java/org/ihtsdo/rvf/execution/service/config/ValidationRunConfig.java (hunk 1 above)
Comment (Tyler-Cooksey) [reply execution-service/src/main/java/org/ihtsdo/rvf/execution/service/config/ValidationRunConfig.java:144]: Fixed.

File: api/src/main/java/org/ihtsdo/rvf/messaging/ValidationQueueManager.java (hunk 2 above)
Comment (Tyler-Cooksey) [reply api/src/main/java/org/ihtsdo/rvf/messaging/ValidationQueueManager.java]: Fixed.

**Review by CoderMChu — APPROVED** [review]
It looks good now.

**Pull request merged**